docs/conformer.rst

```diff
@@ -381,4 +391,4 @@ Command line script
 *******************
 
 
-.. footbibliography::
```

Comment:
Avoid adding this whitespace

Reply:
@kjelljorner Github's acting weird for me, can you mention the line numbers from where to remove the whitespace from?

Reply:
It says that the repository from which you made the pull request has been deleted, so I can no longer look at your source code, but it seems to me that you are changing a white space at the end of the file, lines 383-384. Generally when you make a pull request you should only change the specific lines and not change other parts of the file unless there is a reason to do so.

Reply:
@kjelljorner I'll recommit the PR ASAP. Bad git practice on my end. Thanks